scala/BUILD

```diff
@@ -0,0 +1,15 @@
+load("//scala:scala_toolchain.bzl", "scala_toolchain")
+
+#call scala toolchain
+scala_toolchain(
+      name = 'scala_toolchain_impl',
+      scalacopts = [],
+      visibility = ["//visibility:public"]
+    )
+    
+toolchain(
+    name = 'scala_toolchain',
+      toolchain_type = '@io_bazel_rules_scala//scala:scala_toolchain.bzl',
```

Comment:
toolchain_type should be a label to a target (generally a target of rule toolchain_type()), not an extension.

Reply:
@katre maybe the [docs](https://docs.bazel.build/versions/master/toolchains.html) should contain a snippet of `my_toolchain_type` so that implementors can see an example?

Reply:
Yes, that's a good idea, I'll add it soon.

Reply:
thanks! concretely given the lack of docs, any chance for a small snippet here? so the devs can follow up on your comment

Reply:
sure:

in scala/BUILD, add:
```
toolchain_type(name = 'toolchain_type')
```

Then use '@io_bazel_rules_scala//scala:toolchain_type' as the toolchain_type parameter everywhere.